snomed/com.b2international.snowowl.snomed.datastore.server/src/com/b2international/snowowl/datastore/server/snomed/SnomedModuleDependencyCollectorService.java

```diff
@@ -401,20 +358,4 @@ private LongKeyLongMap getConceptModuleMapping() {
 		return getConfiguration().getConceptModuleMapping();
 	}
 
-	private Collection<SnomedModuleDependencyRefSetMember> getMembers() {
-		return getConfiguration().getMembers();
-	}
-	
-	private Multimap<Long, Long> getModuleMapping() {
```

Comment:
If this property is not used at all then please remove it from ModuleCollectorConfiguration